uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-popup/details-tab/sickness.html

```diff
@@ -4,21 +4,27 @@
     <div class="form-group">
       <label class="col-sm-4 control-label required-mark">Reason</label>
       <div class="col-sm-8">
-        <div class="crm_custom-select crm_custom-select--medium">
-          <select
-            class="no-select2"
-            name="reason"
-            ng-disabled="!detailsTab.isRole('admin') &&
+        <div class="row">
```

Comment:
You don't need a row here. You need a column. Because unlike how it is done in TOIL, this markup has already a bootstrap column with width 4. So instead of adding a column 8, then a row with 11/12 + 1/0 columns, you can just add columns 7/8 + 1/0:

```html
<label class="col-sm-4 control-label required-mark">Reason</label>
<div ng-class="{ 'col-sm-7': detailsTab.showSicknessOptionsEditorIcon, 'col-sm-8': !detailsTab.showSicknessOptionsEditorIcon }">...</div>
<div class="col-xs-1" ng-if="detailsTab.showSicknessOptionsEditorIcon">...</div>
```

However, there is a bit more. In mobile view you want the label be a separate line, which is already like that, because `col-sm-4` will be a full width column in mobile (because of `sm`). But the selector and a wrench should stay in a same line. So you should add rules for mobile:

```html
<!-- this stays as is -->
<label class="col-sm-4 control-label required-mark">Reason</label>
<!-- this stays as is for non-mobile, but for mobile it should still be one line (when wrench is presented), remember, the label ^ will be a separate line in mobile view -->
<div ng-class="{ 'col-sm-7 col-xs-11': detailsTab.showSicknessOptionsEditorIcon, 'col-sm-8': !detailsTab.showSicknessOptionsEditorIcon }">...</div>
<div class="col-xs-1" ng-if="detailsTab.showSicknessOptionsEditorIcon">...</div>
```

Please make sure to update 4 gifs/screenshots:
- admin non-mobile
- admin mobile (yes, we do not support admin for mobile overall, but such components should support mobile from the start)
- ssp non-mobile
- ssp mobile